src/runtime/tests/manifest-test.ts

```diff
@@ -3133,7 +3143,7 @@ resource NobIdJson
     },
     "locations": {}
   }
-`);
+`, {ramDiskMemoryProvider: new RamDiskMemoryImpl()});
```

Comment:
If we were using the new storage stack and want ram disk to be operational, we need to do this (pass the provider as an option) whenever we're parsing manifests.

Right now we aren't using it, and Travis passes as do storageNG tests, and as we encounter places where we're missing this when we look to switch over we'll get an informative exception, so I think it's potentially ok to land this as an initial step and sort out passing `runtime` as the `ramDiskMemoryProvider` on `Manifest.parse` more wholly later, but open to thought.